libs/design-system-components/src/components/ExtensionEditStep2Form/DocumentationLinks/link-element.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
shouldn't this component be in the ds-core? and also made more generic

Reply:
yes, this could be improved, to be transformed in a more generic component, although I would create a separate task for it

Reply:
@josenriagu the link component we have in DS performs a different role